src/components/Profile/components/SchedulePanel/components/ScheduleCalendar/ScheduleCalendar.scss

```diff
@@ -1,3 +1,8 @@
+// @TODO CSSMODULES - Schedule Calendar needs work but left as normal for now
+// Cannot convert to CSS Module because this is using a third party library
+// These styles must not be converted by CSS Modules because the classes they are
+// selecting/overiding are not converted (added afterwards)
```

Comment:
Refactoring Schedule Calendar is a big project outside the scope of this PR, but it's very true that a lot of it needs to be fixed/updated.